src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Application {
+    private static final int arrayLenght = 6;
+    private static FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figureArray = new Figure[arrayLenght];
+
+        for (int i = 0; i < arrayLenght / 2; i++) {
+            figureArray[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = arrayLenght / 2; i < arrayLenght; i++) {
+            figureArray[i] = figureSupplier.getDefaultFigure();
+        }
+
+        for (Figure figure : figureArray) {
+            System.out.println(figure.getFigureData());
```

Comment:
Instead of directly using 'System.out.println', it's better to create a method in the 'Figure' class that will print the figure data. This adheres to the single responsibility principle.